src/pybind/mgr/dashboard/openapi.yaml

```
@@ -12760,6 +12760,28 @@ paths:
       - jwt: []
       tags:
       - RgwRealm
+  /api/rgw/realm/get_valid_realms:
```

```suggestion
  /api/rgw/realms?valid=true
```
Perhaps this can be something like this IMO. But would like to learn more about valid realms definition first.